Dockerfile

```diff
@@ -9,7 +9,9 @@ ARG ARG_USER=app
 
 RUN useradd -d /home/$ARG_USER -m --shell /bin/false --user-group $ARG_USER
 
-RUN apt-get update && apt-get install -q -y --no-install-recommends build-essential git \
+RUN sed -i '/jessie-updates/d' /etc/apt/sources.list  # Now archived
```

Comment:
Instead of a `RUN`, can we vendor the `sources.list` into this repository and `COPY` it into place?